examples/ChatQnA/deploy/k8s_helm.md

```diff
@@ -0,0 +1,497 @@
+     -F 'link_list=["https://opea.dev"]'
+```
+
+This command updates a knowledge base by submitting a list of HTTP links for processing.
+
+Also, you are able to get the file list that you uploaded:
+
+```
+curl -X POST "http://localhost:6007/v1/dataprep/get_file" \
+     -H "Content-Type: application/json"
+
+```
+
+To delete the file/link you uploaded you can use the following commands:
+
+#### Delete link
+```
+# The dataprep service will add a .txt postfix for link file
+
+curl -X POST "http://localhost:6007/v1/dataprep/delete_file" \
+     -d '{"file_path": "https://opea.dev.txt"}' \
+     -H "Content-Type: application/json"
+```
+
+#### Delete file
+
+```
+curl -X POST "http://localhost:6007/v1/dataprep/delete_file" \
+     -d '{"file_path": "what_is_opea.pdf"}' \
+     -H "Content-Type: application/json"
+```
+
+#### Delete all uploaded files and links
+
+```
+curl -X POST "http://localhost:6007/v1/dataprep/delete_file" \
+     -d '{"file_path": "all"}' \
+     -H "Content-Type: application/json"
+```
+
+
+
+## Launch UI
+### Basic UI
+To access the frontend, open the following URL in your browser: 
+`http://{k8s-node-ip-address}:${port}`
+You can find the NGINX port using the following command:
+```bash
+kubectl get service chatqna-nginx
+```
+Which shows the Nginx port as follows:
+```
+NAME            TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)        AGE
+chatqna-nginx   NodePort   10.201.220.120   <none>        80:30304/TCP   16h
+```
+We can see that it is serving at port `30304` based on this configuration via a NodePort.
+
+Next step is to get the `<k8s-node-ip-address>` by running:
+```bash
+kubectl get nodes -o wide
+```
+The command shows internal IPs for all the nodes in the cluster:
+```
+NAME       STATUS   ROLES           AGE   VERSION   INTERNAL-IP    EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION       CONTAINER-RUNTIME
+minikube   Ready    control-plane   11d   v1.31.0   190.128.49.1   <none>        Ubuntu 22.04.4 LTS   5.15.0-124-generic   docker://27.2.0
+```
+When using a NodePort, all the nodes in the cluster will be listening at the specified port, which is  `30304` in this example. The `<k8s-node-ip-address>` can be found under INTERNAL-IP. Here it is `190.128.49.1`.
+
+Open a browser to access `http://<k8s-node-ip-address>:${port}`.
+From the configuration shown above, it would be `http://190.128.49.1:30304`
+
+Alternatively, You can also choose to use port forwarding as shown previously using:
+```bash
+kubectl port-forward service/chatqna-nginx 8080:80
+```
+and open a browser to access `http://localhost:8080`
+ 
+ Visit this [link](https://opea-project.github.io/latest/getting-started/README.html#:~:text=tei%2Dembedding%2Dserver%20%20%20%20%20%20%20%20%20%7C-,Interact%20with%20ChatQnA,-%C2%B6) to see how to interact with the UI. 
+
+### Stop the services
+Once you are done with the entire pipeline and wish to stop and remove all the containers, use the command below:
```

Comment:
Helm removes everything installed for ChatQnA Helm chart, not just containers.

PS.  Helm does not remove (or update) CRDs added by Helm chart, but happily ChatQnA does not have them currently, so everything **is** removed.